app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :posts
```

Comment:
As required, kindly add a foreign key to your user-post association😉.
```suggestion
  has_many :posts, foreign_key: 'author_id'
```